src/pending_message.cpp

```diff
@@ -103,29 +120,8 @@ std::string PendingMessage::ApplyTextInsertingCommands(std::string input, uint32
 		const auto ch = *iter;
 		++iter;
 
-		if (ch == 'N' || ch == 'n') {
-			auto parse_ret = Game_Message::ParseActor(iter, end, escape_char, true);
-			iter = parse_ret.next;
-			int value = parse_ret.value;
-
-			const auto* actor = Main_Data::game_actors->GetActor(value);
-			if (!actor) {
-				Output::Warning("Invalid Actor Id {} in message text", value);
-			} else{
-				output.append(ToString(actor->GetName()));
-			}
-
-			start_copy = iter;
-		} else if (ch == 'V' || ch == 'v') {
-			auto parse_ret = Game_Message::ParseVariable(iter, end, escape_char, true);
-			iter = parse_ret.next;
-			int value = parse_ret.value;
-
-			int variable_value = Main_Data::game_variables->Get(value);
-			output.append(std::to_string(variable_value));
-
-			start_copy = iter;
-		}
+		output.append(cmd_fn(ch, &iter, end, escape_char));
```

Comment:
Instead of hardcoding n and v a callback function is used. This allows easy extending with new command codes